build.gradle

```diff
@@ -514,7 +515,6 @@ configure(project.fineractJavaProjects) {
                     "EqualsGetClass",
                     "EqualsUnsafeCast",
                     "DoubleBraceInitialization",
-                    "UnnecessaryDefaultInEnumSwitch",
```

Comment:
why do you want this?

Reply:
UnnecessaryDefaultInEnumSwitch does not make sense to cause the build to fail. Most of the time it is even very useful eg. to prepare to the enums being extended.